code/modules/organs/internal/species/tajara.dm

```diff
@@ -0,0 +1,125 @@
+/obj/item/organ/internal/eyes/night
+	name = "light sensitive eyes"
+	desc = "A pair of Tajaran eyes accustomed to the low light conditions of Adhomai."
+	icon_state = "tajaran_eyes"
+	action_button_name = "Activate Low Light Vision"
+	relative_size = 8
+	var/night_vision = FALSE
+	var/datum/client_color/vision_color = /datum/client_color/monochrome
+	var/eye_emote = "'s eyes dilate!"
+
+/obj/item/organ/internal/eyes/night/Destroy()
+	disable_night_vision()
+	. = ..()
+
+/obj/item/organ/internal/eyes/night/removed(var/mob/living/carbon/human/target)
+	. = ..()
+	disable_night_vision()
+
+/obj/item/organ/internal/eyes/night/replaced()
+	. = ..()
+	disable_night_vision()
+
+/obj/item/organ/internal/eyes/night/refresh_action_button()
+	. = ..()
+	if(.)
+		action.button_icon_state = "night_eyes"
+		if(action.button)
+			action.button.update_icon()
+
+/obj/item/organ/internal/eyes/night/attack_self(var/mob/user)
+	. = ..()
+	if(owner.last_special > world.time)
+		return
+
+	if(owner.stat || owner.paralysis || owner.stunned || owner.weakened)
+		return
+
+	if(is_broken())
+		return
+
+	if((status & ORGAN_ROBOT))
```

Comment:
what's this for and why? if this is fine, the double parenthesis are useless

Reply:
This is to stop robotic eyes from using this skill.